editoast/editoast_schemas/src/rolling_stock.rs

```diff
@@ -60,22 +61,25 @@ pub struct RollingStock {
     pub locked: bool,
     pub effort_curves: EffortCurves,
     pub base_power_class: Option<String>,
-    /// In m
-    pub length: f64,
-    /// In m/s
-    pub max_speed: f64,
-    pub startup_time: f64,
-    /// In m/s²
-    pub startup_acceleration: f64,
-    /// In m/s²
-    pub comfort_acceleration: f64,
+    #[serde(with = "meter")]
```

Comment:
`"meter"` seems like magic... I wonder if we make more explicit that `meter` is actually a module's path, it would help reduce the magic effect? (that's a general problem of `*` imports, since a simple 'search' in the file for occurrence of `meter` won't return any result, and probably `rust-analyzer` won't help either to get to the definition, it's hard to understand what is going on).

```suggestion
    #[serde(with = "editoast_common::units::meter")]
```

I understand it will make all the macro annotation more noisy, but to me, it would be better. What do you think?

Reply:
https://github.com/OpenRailAssociation/osrd/pull/9928/commits/791877dbaa81e61182489fbbb1403374c129bc2e here you go